src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -109,48 +106,167 @@ class boss_gluth : public CreatureScript
             {
                 switch (eventId)
                 {
-                    case EVENT_WOUND:
-                        DoCastVictim(SPELL_MORTAL_WOUND);
-                        events.ScheduleEvent(EVENT_WOUND, 10000);
-                        break;
-                    case EVENT_ENRAGE:
-                        /// @todo Add missing text
-                        DoCast(me, SPELL_ENRAGE);
-                        events.ScheduleEvent(EVENT_ENRAGE, 15000);
-                        break;
-                    case EVENT_DECIMATE:
-                        /// @todo Add missing text
-                        DoCastAOE(SPELL_DECIMATE);
-                        events.ScheduleEvent(EVENT_DECIMATE, 105000);
-                        break;
-                    case EVENT_BERSERK:
-                        DoCast(me, SPELL_BERSERK);
-                        events.ScheduleEvent(EVENT_BERSERK, 5*60000);
-                        break;
-                    case EVENT_SUMMON:
-                        for (int32 i = 0; i < RAID_MODE(1, 2); ++i)
-                            DoSummon(NPC_ZOMBIE, PosSummon[rand32() % RAID_MODE(1, 3)]);
-                        events.ScheduleEvent(EVENT_SUMMON, 10000);
+                case EVENT_WOUND:
```

Comment:
msvc is wrong when formatting switches, just insert another 4 spaces and it should be fine.